contracts/ProjectValidation.sol

```diff
@@ -0,0 +1,180 @@
+pragma solidity ^0.4.10;
+
+import "zeppelin-solidity/contracts/token/SimpleToken.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+
+
+contract ProjectValidation {
+
+    enum Stages {
+        collectingSignatures,
+        checkerWork,
+        readyToComplete,
+        closing,
+        successfullyClosed,
+        unsuccessfullyClosed
+    }
+
+    struct Checker {
+        address addr;
+        bool signature;
+        bool signed;
+        bool presence;
+    }
+
+    address public starter;
+
+    address public manager;
+
+    Checker public checker;
+
+    address public exchangerContract;
+
+    bool public projectClosed;
+
+    bool public projectCompleted;
```

Comment:
as long as you have notion of state, you don't need this two bool vars, please use state instead in ifs